keras/export/export_lib.py

```diff
@@ -142,13 +142,28 @@ def track(self, resource):
         if isinstance(resource, Layer):
             # Variables in the lists below are actually part of the trackables
             # that get saved, because the lists are created in __init__.
-            self._tf_trackable.variables += resource.variables
-            self._tf_trackable.trainable_variables += (
-                resource.trainable_variables
-            )
-            self._tf_trackable.non_trainable_variables += (
-                resource.non_trainable_variables
-            )
+            if backend.backend() == "jax":
+                self._tf_trackable.variables += tf.nest.flatten(
```

Comment:
Should this be `+=`, or just `=`?

Reply:
The trackable.variables are defined as empty lists in the __init__, so `+=` appends them to the list.